controller/userController.js

```diff
@@ -0,0 +1,34 @@
+const { v4: uuidv4 } = require("uuid");
+
+const { USER_CONTROLLER } = require("../utils/constant");
+const File = require("../models/file");
+const logger = require("../logger/createLogger");
+
+const fileUploader = async (req, res) => {
+  if (!req.file) {
+    return res.json({ error: USER_CONTROLLER.FILE_UPLOAD_ERROR_MSG });
+  }
+
+  // store to database
+  const file = new File({
+    filename: req.file.filename,
+    uuid: uuidv4(),
+    path: req.file.path,
+    size: req.file.size,
+  });
+
+  const response = await file
+    .save()
+    .then((data) => {
+      res.status(200).json({ data });
+    })
+    .catch((err) => {
+      logger.error(
+        `error occured while adding data:${JSON.stringify(err.err)}`,
+      );
+    });
+  logger.info(response);
+  return 0;
```

Comment:
Why returning 0?

Reply:
eslint giving error as
expected a constant value as return in async arrow function
that's why